_docs/setup-collator-node.md

```diff
@@ -7,12 +7,16 @@ tags: [infra, collator]
 
 ## Node Infrastructure Requirements
 On our path to decentralization and trustlessness, collators are a very important piece of the puzzle. We must maintain a high level of performance (as close to a 12 second block production time as possible), and a high degree of availability. As such, the Foundation must prescribe certain technical specifications when onboarding node operators. 
-Bare metal over cloud, especially when costs are a consideration to the node operator.
-- High internet connection (5 gigabits per second).
-- At least 8 CPU cores (fastest core speed).
-- At least 16GB of RAM.
-- At least 1TB of storage.
-- Monitoring and alerting in place (Prometheus, Grafana, the like).
+
+- Network: 1Gbit/s
+- CPU: 4.2GHz
+    - We use Intel Xeon E-2388G processors
+- RAM: 16GB
```

Comment:
We run with 32GB (down from 64GB in AWS). Not sure if the recommendation should be bumped to that.

Reply:
Not particularly. I don't see a reason to bump it unless we want to use our servers as the baseline.